SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Policy
+
+Security is very important for Starlette. 🔒
+
+Learn more about it below. 👇
+
+## Versions
+
+The latest version of Starlette is supported.
+
+You are encouraged to write tests for your application, and update your Starlette version frequently after ensuring that your tests are passing.
+This way you will benefit from the latest features, bug fixes, and **security fixes**.
+
```

Comment:
```suggestion
```

Reply:
Agree with dropping this.
Really all we want to do here is outline a process for reporting potential security issues.

Reply:
committed. 🙏